### ISSUE TITLE
Reports not supported

### ISSUE THREAD

**Issue by bryanhun** [fillup/walmart-partner-api-sdk-php#24]
Any plans to support reports (for items)?

https://developer.walmartapis.com/#get-report

**Comment by fillup** [comment]
Yes we should definitely add that. Would you like to add it and submit a PR? Otherwise I can try to get to it at some point.

**Comment by justinsnair** [comment]
I am close to getting reports to work but I can't get the report to download.  I can see it is responding with a content-disposition file but no matter what I try, guzzle will not download the file.  I should be able to pass 'save_to' with a path to the guzzle client but that isn't working. Any tips?  If i can get it working I will submit a PR to add it in.

**Comment by realadityayadav** [comment]
I would like to add that.
